### ISSUE TITLE
Breakpoints UI

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/debugger#171]
The breakpoints view now shows the breakpoints with the name of their corresponding temporary files:

![image](https://user-images.githubusercontent.com/591645/68696891-1b48de80-057e-11ea-96e5-f8aba0c82d39.png)

Some points to consider:

- Should the breakpoints be grouped per cell, and show something from the cell UI such as the execution count or some text (the first line of the cell for example), if possible?
- Should the breakpoints for previously dumped cells be shown in the UI, even if these cells are not reachable anymore? It might not be obvious to do so, since the files exist on disk and corresponding breakpoints are set in `ptvsd`. Sometimes we can also map back to the same files (editing followed by successive `Ctrl-Z`).

**Comment by martinRenou** [comment]
What would be nice would be to add the line number and the corresponding line, like the JavaScript debugger does in the Chrome developer tools UI
![breakpoints](https://user-images.githubusercontent.com/21197331/71000961-884f1700-20dc-11ea-8835-0d1fe33d8ec2.png)

**Comment by jtpio** [comment]
Yes, there was a similar thing in the early prototype:

![](https://user-images.githubusercontent.com/591645/61885305-e563e980-aefd-11e9-9a33-15eeacb73f09.gif)

**Comment by jtpio** [comment]
So this could be added as second line.

**Comment by JohanMabille** [comment]
Yes that would be great, since the names of the temporary files are not really meaningful
